scripts/quests/otherAreas/Elderly_Pursuits.lua

```diff
@@ -0,0 +1,181 @@
+-----------------------------------
+-- Elderly Pursuits
+-----------------------------------
+-- Log ID: 4, Quest ID: 75
+-- Despachiaire : !pos 108 -40 -83 26
+-- Rouva        : !pos -16 2 11 230
+-- ???          : !pos -412 0 -362 2
+-----------------------------------
+-- !addquest 4 75
+-----------------------------------
+require('scripts/globals/npc_util')
+-----------------------------------
+local carpentersLandingID = zones[xi.zone.CARPENTERS_LANDING]
+-----------------------------------
+
+local quest = Quest:new(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.ELDERLY_PURSUITS)
+
+quest.reward =
+{
+    item = xi.item.ELEGANT_RIBBON,
+}
+
+local function AllParaDead()
+    for i = 0, 4 do
+        local mob = GetMobByID(carpentersLandingID.mob.PARA_OFFSET + i)
+        if not mob:isDead() then
+            return false
+        end
+    end
+
+    return true
+end
+
+quest.sections =
+{
+    {
+        check = function(player, status, vars)
+            return status == QUEST_AVAILABLE and
+            player:hasCompletedQuest(xi.quest.log_id.OTHER_AREAS, xi.quest.id.otherAreas.SECRETS_OF_OVENS_LOST)
+        end,
+
+        [xi.zone.TAVNAZIAN_SAFEHOLD] =
+        {
+            ['Despachiaire'] =
+            {
+                onTrigger = function(player, npc)
+                    return quest:progressEvent(517)
+                end,
+            },
+
+            onEventFinish =
+            {
+                [517] = function(player, csid, option, npc)
+                    quest:begin(player) -- begin quest
+                    return quest:keyItem(xi.ki.ANTIQUE_AMULET)
```

Comment:
Dont quote me on this, but i dont think onEventFinish needs a `return`
so
```lua
            onEventFinish =
            {
                [517] = function(player, csid, option, npc)
                    quest:begin(player) -- begin quest
                    quest:keyItem(xi.ki.ANTIQUE_AMULET)
                end,
            },

Reply:
Seems safe to quote you, doesn't look like it's necessary - removed 👍🏻

Reply:
I wasn't having luck without the return on quest:keyItem (and had seen it used elsewhere in a return, albeit not within onEventFinish).. I could use `npcUtil.giveKeyItem(player, ...` which works without a return

Reply:
So in the event finish now that I look at it, they don't require a returned value, so the npcUtil function would be the correct path here